src/pages/resources.mdx

```diff
@@ -0,0 +1,35 @@
+import HeroBanner from '@site/src/components/HeroBanner'
+
+<HeroBanner title="Resources" />
+
+Welcome to the Fastify resources page! Here, you can find a collection of useful resources for learning and exploring Fastify.
+
+## Books
+
+If you're looking for an in-depth resource to learn Fastify, consider this book:
+
+- [Accelerating Server-Side Development with Fastify: A comprehensive guide to API development for building a scalable backend for your web apps](https://www.amazon.it/Accelerating-Server-Side-Development-Fastify-comprehensive/dp/1800563582/ref=sr_1_1?__mk_it_IT=%C3%85M%C3%85%C5%BD%C3%95%C3%91&crid=3V6FTJ14DL1VN&keywords=fastify&qid=1697134067&sprefix=fastify%2Caps%2C258&sr=8-1)
```

Comment:
```suggestion
- [Accelerating Server-Side Development with Fastify: A comprehensive guide to API development for building a scalable backend for your web apps](https://www.packtpub.com/product/accelerating-server-side-development-with-fastify/9781800563582)
```

Reply:
Just a question, why do you prefer PacktPub instead of Amazon? I think that most people usually buy books on Amazon.

Reply:
Multiple reasons:

1. Why do you link the italian amazon page, despite devs can be from the whole world. I am German, so why did you not link amazon.de. if i created this PR, would you accept that I would use german Amazon?
2. Not the whole world has amazon. E.g. Iran. By using amazon, it could imply that it is only available through amazon. E.g. books which are published by amazon are not available in regular book shops. This is not the case here, but still.
3. Usually a publisher is the "neutral" source. Maybe with more information than a sales page.
4. What if amazon.it shows suddenly the book as out of stock or not as available in your country? It triggers a google search.
5. With the publisher, I get the info from a neutral source. I know the publisher, the tite and the ISBN. I can order maybe directly from the publisher or search in google where to get it.

Reply:
Got it, thanks! I'll make both changes.